mne/io/edf/edf.py

```diff
@@ -141,10 +139,14 @@ class RawEDF(BaseRaw):
 
         >>> events[:, 2] >>= 8  # doctest:+SKIP
 
-    In addition, for GDF files, the stimulus channel is constructed from the
-    events in the header. The id numbers of overlapping events are simply
-    combined through addition. To get the original events from the header,
-    use function :func:`mne.io.find_edf_events`.
+    The TAL channel called 'EDF Annotations' or 'BDF Annotations' is parsed and
+    put in the raw.annotations attribute.
+    Use :func:`mne.events_from_annotations` to obtain events from the
+    annotations.
+
+    If channels named 'Status' or 'STATUS' will be considered analog stim
+    channels, use method ``raw.find_edf_events`` to recover the events encoded
+    in this channel.
```

Comment:
so we deprecate find_edf_events in the next PR?

Reply:
+1 for deprecating this (in another PR is fine). I think we should also backport this fix if we can, since it really should have been in 0.17

Reply:
(Although given this is +295 −611, we could instead consider deprecating it in 0.18 and make a separate PR to maint/0.17 to fix the message that users currently get, which is probably wrong)

Reply:
I think Depreciation of find_edf_events should be in a different PR. And backported.

Reply:
Is this actually correct, though? It sounds wrong to me. I thought that `Status` channels were just analog channels, and didn't have any relationship to `find_edf_events`. I thought that the `TAL` channels were the ones that ended up putting events in the array returned by `find_edf_events`. No?

Reply:
`find_edf_events` != `events_from_annotations`

Reply:
I don't understand how that answers my question.

I thought that TAL channels are what added events to the array returned by `find_edf_events`, and that Status/STATUS channels were just analog channels. Is the array returned by `find_edf_events` created by doing a `find_events`-like operation on the STATUS channel?

Reply:
> I thought that TAL channels are what added events to the array returned by find_edf_events, and that Status/STATUS channels were just analog channels. Is the array returned by find_edf_events created by doing a find_events-like operation on the STATUS channel?

It only bypasess the annotation reading, and (second-samples) convertion. Which I don't understand 'cos the TAL channel is in seconds already. 

What is wrong here (I think) is the comment. I think it should suggest to do `mne.find_events(raw, stim_channel='Status')`. 

I'm just checking that it does what we would expect

Reply:
> It just returns what ever has been stored in _extras. Which I'm not sure is supported by all EDF files.

This just begst the question: what gets stored in `_raw_extras`? Where do these values come from? IIRC it comes from the TAL channels, not the analog STATUS channel. Looking at the code, [this seems to be the case](https://github.com/mne-tools/mne-python/blob/master/mne/io/edf/edf.py#L340-L346).

So this bit of docs added in this PR:
```
    If channels named 'Status' or 'STATUS' will be considered analog stim
    channels, use method ``raw.find_edf_events`` to recover the events encoded
    in this channel.
```
does not seem to make sense. `find_edf_events` should not be used to recover events encoded in the `STATUS` channel, it should only be used to recover the events that had been in the TAL channels. `find_events` should be used to get data from the analog STATUS channels. Right? If so, you can just delete these lines because getting events using `find_events` from STIM-type channels is standard behavior.

Reply:
yes. That's exactly it, and it is tested. 

```py
    # Find events
    events = find_events(raw, verbose=1)
    events[:, 2] >>= 8  # last 8 bits are system events in biosemi files
    assert_equal(events.shape[0], 2)  # 2 events in file
    assert_array_equal(events[:, 2], [20, 28])
```
Do you think we should make the channel splicit?

Reply:
I'd say either remove this comment or have it just say "If channels named 'Status' or 'STATUS' are found, they will be considered analog stim channels".